app/assets/stylesheets/components/_summary-card-component.scss

```diff
@@ -0,0 +1,7 @@
+.app-c-summary-card-component {
+  .govuk-summary-list__row {
+    .govuk-summary-list__value {
+      width: 30%;
```

Comment:
I don't think there's any need to specify this width: the `key` elements are already given an explicit width so this effectively sets both the `key` and `value` elements to 50%. 

In that case there would be no need to add specific styles at all in addition to what are already provided.

Reply:
i swear it was moving the edit/delete links down for the rows without this but now it seems to be working fine 😂 have removed it 👍